docs/book/src/cronjob-tutorial/testdata/project/api/v1/cronjob_types.go

```diff
@@ -169,11 +169,19 @@ type CronJobStatus struct {
 type CronJob struct {
 	/*
 	 */
-	metav1.TypeMeta   `json:",inline"`
+	metav1.TypeMeta `json:",inline"`
+
+	// metadata is a standard object metadata
+	// +optional
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   CronJobSpec   `json:"spec,omitempty"`
-	Status CronJobStatus `json:"status,omitempty"`
+	// spec defines the desired state of CronJob
```

Comment:
to avoid confusion, I would remove comment in Spec/Status structs (in this case, line 65 and line 145) since they will be overridden by these new comments, in all `api` files

Reply:
The type still with the same desc ( without a dot at the end only )
Then, we added desc for 

```

	Spec   CronJobSpec   `json:"spec,omitempty"`
	Status CronJobStatus `json:"status,omitempty"`
```

That was without any before